app/views/profiles/show_collections_questions.html.erb

```diff
@@ -1,25 +1,80 @@
-<div class="d-flex justify-content-between align-items-center">
-  <h1>Collections Specific Questions and Answers</h1>
-  <%= link_to "Back to Profile", profile_path(current_user), class: "link_to" %>
-</div>
-<% if @collection_answers.present? %>
-  <% @collection_answers.each do |collection, qa_data| %>
-    <div class="d-flex align-items-center gap-3">
-      <h3><%= collection.division %></h3>
-      <%= link_to collection_questions_profile_path(collection), class: 'link_to mb-2' do %>
-          <i class="bi bi-pencil"></i> <span class="fw-light">Edit</span>
+<div class="container py-4">
+  <!-- Header -->
+  <div class="d-flex justify-content-between align-items-center mb-4">
+    <h1 class="h3 mb-0">Loan Questions and Answers</h1>
+    <%= link_to "Back to Profile",
+                profile_path(current_user),
+                class: "btn btn-outline-secondary" %>
+  </div>
+
+  <% if @loan_answers.present? %>
```

Comment:
@loan_answers here are always empty because show_collections_questions method in profile_controller creates only  @collection_answers. @loan_answers are created in show_loan_questions method. You can copy the code from show_loan_questions to show_collections_questions and delete show_loan_questions, or do any other refactoring